app/workers/mentions/send_email_notification_worker.rb

```diff
@@ -7,7 +7,7 @@ def perform(mention_id)
       mention = Mention.find_by(id: mention_id)
       return unless mention
 
-      NotifyMailer.with(mention: mention).new_mention_email
+      NotifyMailer.with(mention: mention).new_mention_email.deliver_now
```

Comment:
oh - that makes sense - rather than asking for an email, ask for an email _and then deliver it_